lib/validic/response.rb

```diff
@@ -11,5 +13,23 @@ def initialize(summary, response)
         @attributes = response
       end
     end
+
+    def next
+      extend Validic::REST::Utils
```

Comment:
this may require a revamp of what we have now, can we do the refactor on a later version?